### ISSUE TITLE
No album art when playing last.fm through squeezescrobbler

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#36]
```
See summary ;). 

The official Logitech player and the Duet control both show album art correctly.
```

Original issue reported on code.google.com by `arnout.engelen` on 2011-07-30 09:47:01

**Comment by kaaholst** [comment]
```
in the squeezeserver logs I see the following metadata about a song being played, but
this probably doesn't move to the controller in this format:

{
          'nicetime' => '4:04',
          'lastfm:trackauth' => '85aef',
          'creator' => 'Rose Royce',
          'location' => 'http://play.last.fm/user/38a07cff545dfbf51c201a989487d7c6.mp3',
          'lastfm:albumId' => '2124688',
          'duration' => '244000',
          'freeTrackURL' => undef,
          'artistpage' => 'http://www.last.fm/music/Rose+Royce',
          'image' => 'http://userserve-ak.last.fm/serve/174s/64506661.jpg',
          'lastfm:artistId' => '2101',
          'album' => 'Greatest Hits',
          'buyAlbumURL' => undef,
          'title' => 'Love Don\'T Live Here Anymore',
          'id' => '1979348',
          'buyTrackURL' => undef,
          'trackpage' => 'http://www.last.fm/music/Rose+Royce/_/Love+Don%27T+Live+Here+Anymore',
          'albumpage' => 'http://www.last.fm/music/Rose+Royce/Greatest+Hits'
        };
```

Original issue reported on code.google.com by `arnout.engelen` on 2011-07-30 10:03:06

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:15:53Z
